.github/workflows/java_binding_publish.yml

```diff
@@ -0,0 +1,140 @@
+name: Create Java release
+
+on:
+  workflow_dispatch:
+    paths:
+      - .github/workflows/java_binding_publish.yml
+      - bindings/java/**
+      - src/**
+      - Cargo.lock
+      - Cargo.toml
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
This is also not needed I think or maybe not even wanted?

Reply:
Why not? Happy to remove

Reply:
removed